### PR TITLE
Add body layout

### PR DESCRIPTION
**Pull request by marioabreu**

### CODE REVIEW

**Comment by azevedo** [comment]
This has commits from the other PR. It's confusing, I don't know what's new.

File: app/assets/stylesheets/components/_event.sass [hunk 1]
```diff
@@ -1,7 +1,6 @@
-$Event-color-background: Theme-color(background, default)
-$Event-borderTop-color: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
-$Event-color-shadow: Theme-color(background, shadow)
-$Event-color-title: Theme-color(text, default)
+$Event-color-background: Theme-color(background, event-card-color)
+$Event-color-borderTop: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
```
Comment (marioabreu) [inline app/assets/stylesheets/components/_event.sass]: I have to fix this now.

File: app/assets/stylesheets/components/_theme.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@ $Theme-fontSize-large: 1.222rem;
 $Theme-spacing-base: 18px;
 $Theme-spacing-small: 9px;
 
+
```
Comment (marioabreu) [inline app/assets/stylesheets/components/_theme.scss]: I have to remove this empty line.

File: app/assets/stylesheets/components/_theme.scss [hunk 3]
```diff
@@ -13,32 +13,66 @@ $Theme-fontSizes: (
...
+$Theme-lightGray: #D8D1C2;
+$Theme-gray: #B4B4B4;
+
 $Theme-lightGreenSuccess: #E6F0C0;
+$Theme-greenSuccess: #566612;
 $Theme-lightRedError: #FCE3E4;
 $Theme-redDanger: #BF283A;
 $Theme-redError: #981215;
 
+$Theme-margin-left-logo: 20px;
```
Comment (azevedo) [inline app/assets/stylesheets/components/_theme.scss]: Not being used. Besides you can put this on the Header component

File: app/assets/stylesheets/components/_theme.scss [hunk 4]
```diff
@@ -13,32 +13,66 @@ $Theme-fontSizes: (
...
+
 $Theme-color: (
   background: (
-    default: $Theme-white,
     error: $Theme-lightRedError,
-    success: $Theme-lightGreenSuccess,
-    shadow: $Theme-gray
+    event-card-color: $Theme-beige,
```
Comment (azevedo) [inline app/assets/stylesheets/components/_theme.scss]: the suffix `-color` is redundant. you can remove it

Comment (azevedo) [inline app/assets/stylesheets/components/_theme.scss]: Also, talk with @fcbaila in order to rename these keys using our left-to-right convention

File: app/controllers/events_controller.rb [hunk 5]
```diff
@@ -1,7 +1,7 @@
 class EventsController < ApplicationController
   authorize_resource
   def index
-    @events = Event.all
+    @events_per_month = Event.future.group_by { |event| event.start_date.month }
```
Comment (azevedo-252) [inline app/controllers/events_controller.rb]: I would create a scope in the model for retrieving these events

File: app/assets/stylesheets/components/_theme.scss (hunk 3 above)
Comment (marioabreu) [inline app/assets/stylesheets/components/_theme.scss]: :+1:

File: app/views/events/index.slim [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
-        = link_to event_path(event.id), class: 'Event-title' do
-          .Event-title = event.name
-          .Event-info
-            em = "On #{Date::DAYNAMES[event.start_date.wday]}"
-          .Event-info = event.starting_date
-          .Event-info
-            span.fa.fa-user = "#{event.attendees.size}"
+-@events_per_month.each do |month|
+  ul.Grid
+    p =Date::MONTHNAMES[month[0]]
```
Comment (azevedo) [inline app/views/events/index.slim]: Leave a _space_ after the _=_

File: db/seeds.rb [hunk 7]
```diff
@@ -5,11 +5,63 @@
...
-                        {name: 'Javascript-cenas', location: 'Braga', start_date: DateTime.new(2015,03,10,20,00), end_date: DateTime.new(2015,03,10,21,30), description: 'Temporibus, obcaecati aliquam esse magnam veniam illo voluptatum distinctio, quam enim dolore, ipsa et debitis.', hashtag: '#javascipt-cenas', owner: 1},
-                        {name: 'Web-development-cenas', location: 'Braga', start_date: DateTime.new(2015,03,10,20,30), end_date: DateTime.new(2015,03,10,22,30), description: 'Vitae explicabo ea praesentium non soluta suscipit temporibus, obcaecati aliquam esse magnam veniam illo voluptatum distinctio, quam enim dolor.', hashtag: '#web-cenas', owner: 2},
-                        {name: 'YEY', location: 'Braga', start_date: DateTime.new(2015,03,10,20,30), end_date: DateTime.new(2015,03,10,22,30), description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Vitae explicabo ea praesentium non soluta suscipit temporibus, obcaecati aliquam esse magnam veniam illo voluptatum distinctio, quam enim dolore, ipsa et debitis.', hashtag: '#YEY', owner: 2}])
+binding.pry
```
Comment (azevedo) [inline db/seeds.rb]: Forgotten

**Comment by azevedo** [comment]
:+1:

**Pull request merged**